apps/01PigLatin.js

```diff
@@ -7,11 +7,80 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer:
+  // in order to ensure that there are no weird capitalizations in the middle of the word and that all words have consistent capitalization, String.prototype.toLowerCase() is used make all letters lowercase
+  word  = word.toLowerCase();
 
+  // Concept Check
+  //
+  // What is the significance of setting vowelIndex to -1?
+  // This might help:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+  //
+  // Your Answer:
+  // vowelIndex is set to -1 to show that there is currently no value assigned to the variable, meaning that no vowels have been found
+  var vowelIndex = -1; // Set it to assume there are no vowels
```

Comment:
Not quite. We are initializing **vowelIndex** to -1, so it does have a value assigned to it.

The value -1 indicates that we have not yet found a vowel.